PROJECT

```diff
@@ -4,3 +4,12 @@ layout:
 projectName: odh-model-controller
 repo: github.com/opendatahub-io/odh-model-controller
 version: "3"
+resources:
+- api:
+    crdVersion: v1alpha1
```

Comment:
before GA we need to change this to v1. GA releases should ideally not contain alpha or beta APIs

Reply:
The version is changed to v1.